src/components/Navbar/Navbar.view.tsx

```diff
@@ -1,40 +1,26 @@
-import React, { Dispatch, SetStateAction } from 'react';
 import { NavLink } from 'react-router-dom';
 import LogoutIcon from '@mui/icons-material/Logout';
-import { Box, SxProps } from '@mui/material';
-import { Theme } from '@mui/material/styles';
+import { Box } from '@mui/material';
 import IconButton from 'components/IconButton';
 import { MenuItem } from 'types';
+import Styled from './Navbar.styled';
 
 interface Props {
-    sx?: SxProps<Theme>;
     activePage: string;
-    setActivePage: Dispatch<SetStateAction<string>>;
+    onPageSelect: (name: string) => void;
     menuItems: MenuItem[];
     logout: () => void;
+    className?: string;
```

Comment:
You could probably create Props once and export it for Mobile and View usages.

Reply:
Done